roles/overpass-api/tasks/set_up_database.yml

```diff
@@ -0,0 +1,15 @@
+- name: install osmctools
+  apt:
+    pkg: osmctools
+
+- name: download planet pbf
+  get_url:
+    url: "{{ planet_pbf_url }}"
+    dest: "{{ database_dir }}/planet-latest.osm.pbf"
+  become: true
+  become_user: "{{ user }}"
+
+- name: import planet
+  shell: osmconvert {{ database_dir }}/planet-latest.osm.pbf --out-osm | update_database --db-dir={{ database_dir }} --meta
+  become: true
+  become_user: "{{ user }}"
```

Comment:
Good idea to import the planet, but it shouldn't be re-imported if it was already imported.
It might also a bit too long for an ansible script - I'm not sure if ansible/ssh is robust enough to execute commands that can last several hours.